### DISCUSSION TITLE
Live server is not responding.

### DISCUSSION THREAD

**Discussion by SKSameerAhmad1** [coder/code-server#6009] [Q&A]
I write HTML code in PW labs (in vs code), than live server is not responding.
And when I write code in Normal VS code than live server is responding... why?

**Comment by code-asher** [comment]
Is this the ms-vscode.live-server extension?

I tried it and the internal preview seems to work for me.  I do 
get errors like "failed to load message bundle" though.  It looks 
like it might be making non-relative requests to the root which 
will not work because we proxy it through a sub-path.  If I am 
right about that this is something that will need to be fixed in 
the extension.

I am also seeing a web socket get blocked due to mixed content. 
That also seems like it would need to be fixed in the extension.

Lastly I tried in Codespaces to see if it has the same problem but 
there the iframe gets completely blocked.
